packages/openapi-generator/test/test-config-wrong-key.json

```diff
@@ -0,0 +1,5 @@
+{
+  "input": "some-repository",
```

Comment:
[q] Should this and the other test config be actual files or could we use mock-fs?

Reply:
Since we use `yargs` now the mock did not work. Seems yargs uses not the `fs` module to lead the file or spawns a different process to load the file which also block the mock.